docs/user-guide/cli-using-using-daemon-mode.md

```diff
@@ -42,7 +42,7 @@ To enable daemon mode and configure Zowe to run Zowe CLI constantly in daemon mo
 
    You have successfully configured Zowe CLI to run on daemon mode.
    
-   Each time a Zowe CLI command is issued, the daemon binary creates or reuses a file in the user's home directory. In some cases, this behavior might be undesirable. (For example, the home directory resides on a network drive and has poor file performance.)
```

Comment:
@anaxceron, I think this example (line 45) is useful. However, the example from line 49-58 could be removed.

Reply:
Resolved w/ [PR 3479](https://github.com/zowe/docs-site/pull/3479)